lib/repl.js

```diff
@@ -928,6 +951,16 @@ function complete(line, callback) {
   function completionGroupsLoaded(err) {
     if (err) throw err;
 
+    if (warning.type) {
+      self._writeToOutput('\r\n');
+      process.emitWarning(
+        warning.message,
+        warning.type,
+        undefined,
+        undefined,
+        true);
```

Comment:
There is no boolean fifth argument to `process.emitWarning()` so I'm not sure what this is doing.